ci/submodule-sync.sh

```diff
@@ -89,6 +90,18 @@ else
   echo "Test failed, will update the result"
 fi
 
+# Extract the rapids-cmake sha1 that we need to pin too
+rapids_cmake_sha=$(git -C thirdparty/cudf/cpp/build/_deps/rapids-cmake-src/ rev-parse HEAD)
+echo "Update rapids-cmake pinned SHA1 to ${rapids_cmake_sha}"
+echo "${rapids_cmake_sha}" > thirdparty/cudf-pins/rapids-cmake.sha
+
+# Do the git add after the build so that we get
+# the updated versions.json generated by the build
+echo "Update cudf submodule to ${cudf_sha} with updated pinned versions"
+git add .
+git diff-index --quiet HEAD || git commit -s -m "Update submodule cudf to ${cudf_sha}"
```

Comment:
There was already a git add and commit performed above.  That block above was removed in the reference version of this change.

Reply:
oh, let me remove above one.
I'll use cherry-pick way to force push the branch and update this PR.

Reply:
build/submodule-check will fail if you don't `git add` the submodule, it shouldn't have to be committed.

Reply:
Let me run a test against it, if so, we just need 2 git add and one git commit at last.

Reply:
Thanks, @jlowe. It works. I remove the first commit line.